miner_utils/upload_model.py

```diff
@@ -152,14 +153,14 @@ async def main(config: bt.config):
     # We can only commit to the chain every n minutes, so run this in a loop, until successful.
     while True:
         try:
+            await model_metadata_store.store_model_metadata(
+                wallet.hotkey.ss58_address, model_id_with_hash
+            )
```

Comment:
Why is this part moved here?

Reply:
I think the idea was to not make the repo public until after posting the repo info to the chain to further assist with trying to evade the copycats